test_assets/pkcs12/leaf_password_is_abc.parsed.bin

```diff
@@ -0,0 +1,60 @@
+Pfx {
```

Comment:
Since this is the Debug representation and not actual binary, maybe the extension should be `.txt`. What do you think?

Reply:
Yep, I agree

Reply:
Done